README.md

```diff
@@ -31,77 +51,154 @@ pathways. We anticipate that **SCT** will bring together the spinal cord
 neuroimaging community by establishing standard templates and analysis
 procedures.
 
+## Installation
+
+### Install from package (release)
+
+The simplest way to install SCT is to do it via a stable release. First, download the [latest release](https://github.com/neuropoly/spinalcordtoolbox/releases/latest). Major changes to each release are listed [here](CHANGES.md).
+
+N.B. We currently cover OS X (10.8+) and Debian/Ubuntu/Fedora/RedHat/CentOS platforms. If you have another OS, please use a [virtual machine](https://github.com/neuropoly/spinalcordtoolbox/tree/jca_readme#using-virtual-machine).
+
+If you have 10.7 or less, the only incompatibility is with ANTs, which you can compile on your station and then copy the binaries under SCT installation as explained [here](https://github.com/neuropoly/spinalcordtoolbox/wiki/binaries).
+
+Once you have downloaded SCT, unpack it (note: Safari automatically unzip it). Then, open a new Terminal, go into the created folder and launch the installer:
+
+~~~
+./install_sct
+~~~
+
+### Install from Github (development)
+
+Download source code from Github (N.B. you need to have git installed):
+
+~~~
+git clone --depth=1 --branch=master https://github.com/neuropoly/spinalcordtoolbox.git sct
+~~~
+
+Install SCT:
+
+~~~
+cd sct
+./install_sct
+~~~
+
+Note: To prevent selective download of data, python or binaries, the following flags can be used:
```

Comment:
Easy to shoot oneself in the foot without further explanations (eg. you can get used to not downloading python or data, and then BAM a template or a dependency was updated and then you cry).

Reply:
So maybe this note could disappear from the concise README, but could exist in the Sphinx docs, with the associated caveats.